--- ./plugins/inputs/all/all.go	2020-09-30 16:18:03.877962580 +0530
+++ ./plugins/inputs/all/all-new.go	2020-09-30 16:28:19.713980088 +0530
@@ -38,6 +38,7 @@
 	_ "github.com/influxdata/telegraf/plugins/inputs/docker_log"
 	_ "github.com/influxdata/telegraf/plugins/inputs/dovecot"
 	_ "github.com/influxdata/telegraf/plugins/inputs/ecs"
+	_ "github.com/influxdata/telegraf/plugins/inputs/eis_msgbus"
 	_ "github.com/influxdata/telegraf/plugins/inputs/elasticsearch"
 	_ "github.com/influxdata/telegraf/plugins/inputs/ethtool"
 	_ "github.com/influxdata/telegraf/plugins/inputs/eventhub_consumer"
@@ -81,7 +82,6 @@
 	_ "github.com/influxdata/telegraf/plugins/inputs/kinesis_consumer"
 	_ "github.com/influxdata/telegraf/plugins/inputs/kube_inventory"
 	_ "github.com/influxdata/telegraf/plugins/inputs/kubernetes"
-	_ "github.com/influxdata/telegraf/plugins/inputs/lanz"
 	_ "github.com/influxdata/telegraf/plugins/inputs/leofs"
 	_ "github.com/influxdata/telegraf/plugins/inputs/linux_sysctl_fs"
 	_ "github.com/influxdata/telegraf/plugins/inputs/logparser"
